pre-build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
I do not have anything against this prebuild script but the main answer is the `why` and `whatfor`

IMO we need to:
 - Update the doc/READ with clear intructions on running tests
 - Mention that prebuild script potentially allow running side service tests against arbitrary Sabre server.